sos/plugins/openstack_aodh.py

```diff
@@ -72,8 +88,21 @@ def postproc(self):
 
         self.do_file_sub(
             "/etc/aodh/aodh.conf",
-            r"(auth_url[\t\ ]*=[\t\ ]*)(.+)",
+            r"(rabbit_password[\t\ ]*=[\t\ ]*)(.+)",
```

Comment:
Isnt this a regression that auth_url value won't be obfuscated?

Or have there been a need to scrub auth_url value at all?

Reply:
we should not obfuscate the auth_url. the auth_url does not password information in and for trouble shooting it is useful to know which endpoint we connect to. E.g. this is an example of an auth_url:

auth_url=http://172.17.1.19:5000

Reply:
Are you certain of that? Generally, an HTTP URL may include both username and password for HTML basic auth. If we can be absolutely certain that will never be the case here then this is OK (it would also be better if it did not elide the entire URL, but rather only actual auth elements, if present).

Reply:
This is not a basic auth or so. You have different options for that. This information is used to query a token from keystone. The User information is passed via json of the POST request.

# Authentication URL (string value)
#auth_url = <None>
# User id (string value)
#user_id = <None>
# User's password (string value)
#password = <None>

Reply:
As long as users aren't likely to modify that, it's fine (we have had other URLs that we've ended up having to obfuscate in the past for this).